glib/Cargo.toml

```diff
@@ -22,6 +22,7 @@ name = "glib"
 once_cell = "1.0"
 libc = "0.2"
 bitflags = "1.0"
+fnv = "1.0.7"
```

Comment:
Maybe https://crates.io/crates/rustc-hash would be a better choice? Claims to be faster than FVN, is used in rustc and maintained by the Rust team.

Reply:
Oh! Totally missed it. Giving it a try now. Thanks.

Reply:
nnethercote wrote a very nice blog about why it was faster [here](https://nnethercote.github.io/2021/12/08/a-brutally-effective-hash-function-in-rust.html). I'm not sure if it's "valid" for all cases.

Reply:
Interesting blog post. Thanks @GuillaumeGomez

Reply:
With `rustc-hash`, callgrind shows 70% reduction in cycle estimation for `panicked` compared to the hasher from `std` (50% for `fnv`). I observe similar CPU usage and stats to those obtained with `fnv` for `ts-standalone`.

Reply:
Let's go with that one then, thanks!